tests/loki/directbeam_test.py

```diff
@@ -32,6 +38,7 @@ def test_can_compute_direct_beam_for_all_pixels():
     )
     providers = loki_providers()
     pipeline = sciline.Pipeline(providers, params=params)
+    pipeline.set_param_series(PixelMaskFilename, [])
```

Comment:
Is it possible to have a default param series so one does not have to specify this every time? (I mean like the `default_parameters`). I guess not...

Reply:
Not possible currently, but this is changing anyway with the Sciline rewrite.